src/main/java/org/folio/consortia/service/impl/PublicationServiceImpl.java

```diff
@@ -246,11 +246,8 @@ private PublicationStatusEntity savePublicationStatusRecord(int totalRecords) {
     publicationStatusEntity.setId(UUID.randomUUID());
     publicationStatusEntity.setStatus(PublicationStatus.IN_PROGRESS);
     publicationStatusEntity.setTotalRecords(totalRecords);
-
-    var savedPSE = publicationStatusRepository.save(publicationStatusEntity);
-
-    log.info("Created publication record {}", savedPSE.getId());
-    return savedPSE;
+    log.info("Created publication record {}", publicationStatusEntity.getId());
+    return publicationStatusEntity;
```

Comment:
THis method now does not save anything, just create PublicationStatusEntity , so need to remane in some way

Reply:
Changed